Source/CombatExtended/Harmony/Harmony_JobGiver_OptimizeApparel.cs

```diff
@@ -0,0 +1,33 @@
+﻿using HarmonyLib;
+using RimWorld;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using Verse;
+using Verse.AI;
+
+namespace CombatExtended.HarmonyCE
+{
+    /* Overrides the CompReloadable reload job if the pawn has suitable ammo in their inventory.
```

Comment:
You copied this patch class from another file but didn't update the description.